tests/examples/source/devfiles/nodejs/devfile-with-valid-events.yaml

```diff
@@ -20,6 +20,22 @@ components:
       mountSources: true
       memoryLimit: 1024Mi
 commands:
+  - exec:
+      id: myPreStart
+      commandLine: echo I am a myPreStart
```

Comment:
Let's do something like `echo hello test 1 >> $PROJECTS_ROOT/test.txt` or something like that and check that this string is present two times in the test.txt and the other string is present 1 time only.